lib/config/virtual_container_host.go

```diff
@@ -214,6 +216,16 @@ func (t *VirtualContainerHostConfigSpec) SetMoref(moref *types.ManagedObjectRefe
 	}
 }
 
+// SetIsCreating sets the ID of the VCH to a constant, to identify the creating VCH VM before the VM moref can be set into this property
+func (t *VirtualContainerHostConfigSpec) SetIsCreating() {
```

Comment:
You have a SetIsCreating() but no offsetting function.  Once you set it to creating, it's set like that for good?